drivers/fuel_gauge/max17048/max17048.c

```diff
@@ -90,7 +90,7 @@ int max17048_voltage(const struct device *i2c_dev, uint16_t *response)
 	 * milli volts instead of volts.
 	 */
 
-    *response = (uint16_t)((uint32_t)*response * 78125L / 1000000L);
+	*response = (uint16_t)((uint32_t)*response * 78125L / 1000000L);
```

Comment:
By the way. We also make use of `clang-format` I highly recommend using that for formatting your code.

Also in general `ClangD` is great :)

Reply:
Good call. Will do for future work as we are making a substantial bet on Zephyr am I look forward to more contributions. Thanks for your time and guidance.

Reply:
That's great!

If you're interested, you should check out the wiki https://github.com/zephyrproject-rtos/zephyr/wiki. It has links to the working groups as well as a link to the Discord which is really active.